docs/source/challenge_creation.md

```diff
@@ -87,6 +87,7 @@ First of all, open the [`challenge_config.yml`](https://github.com/Cloud-CV/Eval
     * **max_submissions_per_day**: Positive integer which tells the maximum number of submissions per day to a challenge phase.
 
     * **max_submissions**: a Positive integer that decides the overall maximum number of submissions that can be done to a challenge phase.
+    * **monthly_submission_limit_per_team**: Positive integer which tells the maximum number of submissions per month for every single team to a challenge phase.
```

Comment:
Hi @Mike-Dai 
`monthly_submission_limit_per_team` is not a correct field for  `Monthly Submission Limit Per Team`.
Please go through [EvalAI/pull#2005](https://github.com/Cloud-CV/EvalAI/pull/2005).
Please let me know if you have questions. :)